backend/entityservice/logger_setup.py

```diff
@@ -0,0 +1,90 @@
+import os
+import logging.config
+import structlog
+
+import yaml
+
+
+def setup_logging(
+    default_path='logging.yaml',
+    default_level='INFO',
+    env_key='LOG_CFG'
+):
+    """
+    Setup logging configuration
+    """
+    path = os.path.dirname(os.path.abspath(__file__)) + '/' + default_path
+    value = os.getenv(env_key, None)
+    if value:
+        path = value
+    if os.path.exists(path):
+        with open(path, 'rt') as f:
+            config = yaml.safe_load(f.read())
+        logging.config.dictConfig(config)
+    else:
+        print("YAML file doesn't exist")
+        default_level = _str_to_level(default_level)
+        logging.basicConfig(level=default_level)
+
+    # Configure Structlog wrapper for client use
+    setup_structlog()
+
+
+def setup_structlog():
+    structlog.configure(
+        processors=[
+            structlog.stdlib.filter_by_level,
+            structlog.stdlib.add_logger_name,
+            structlog.stdlib.add_log_level,
+            structlog.stdlib.PositionalArgumentsFormatter(),
+            structlog.processors.TimeStamper(fmt='iso'),
+            structlog.processors.StackInfoRenderer(),
+            structlog.processors.format_exc_info,
+
+            # Uncomment ONE Renderer:
+            #structlog.processors.KeyValueRenderer(),
+            #structlog.processors.JSONRenderer(),
+            structlog.dev.ConsoleRenderer()
```

Comment:
Would it make more sense to have this a settable via the config file?

Reply:
Yeah I think making all of structlog configurable would be a good extension - I'll create an issue rather than add it now though.